src/Core/src/Primitives/FlowDirection.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Maui
+{
+	public enum FlowDirection 
```

Comment:
don't we have this above ?

Reply:
Yeah, the one from the Forms repo snuck across. Fixed.